_layouts/base.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
I :heart: this structure, basing both layouts of `base.html`

Reply:
:)